apps/st2-history/template.html

```diff
@@ -113,7 +113,14 @@ <h2 class="st2-flex-table__caption-title">{{ group.period | date:'fullDate':'UTC
                   {{ record.trigger.type }}
                 </span>
               </span>
-              <span ng-switch-default>Manual</span>
+              <span ng-switch-default>
+                <span class="st2-history__column-app-name">
+                  Manual
```

Comment:
The execution could either be created by user creating it though API (manually) or by rule (somewhat "automatic"). I'm actually thinking we should also show the application it was called with (direct API call, st2client, web ui, hubot, etc.), but that would require some additional info to be provided (for example, oAuth-like, by issuing an application token).